src/App.vue

```diff
@@ -166,19 +166,7 @@
         </v-container>
       </v-content>
       <snackbar/>
-      <v-dialog v-model="dialog" width="500">
-        <v-card>
-          <v-toolbar dark>
-            <v-btn icon dark @click="dialog = false">
-              <v-icon>close</v-icon>
-            </v-btn>
-            <v-toolbar-title class="white--text">Create a new team</v-toolbar-title>
-          </v-toolbar>
-          <v-card-text>
-            <CreateTeam @close="dialog = false" />
-          </v-card-text>
-        </v-card>
-      </v-dialog>
+      <CreateTeamModal :dialog="dialog" @modalChanged="changeDialog"></CreateTeamModal>
```

Comment:
1. Rename the component to `CreateTeamDialog`
2. Please use `<create-team-dialog>`
3. `:dialog` prop is not well named, `:show` will be better
4. Use `@close` instead of `@modalChanged`

Reply:
Thanks for the feedback, I have applied the fixes!

Reply:
Great, I added a comment after my tests